cmd/influx/setup.go

```diff
@@ -240,10 +240,14 @@ var (
 	errPasswordIsTooShort = fmt.Errorf("passwords is too short")
 )
 
-func getPassword(ui *input.UI) (password string) {
+func getPassword(ui *input.UI, showNew bool) (password string) {
```

Comment:
If we passed the prompt string instead of using a boolean flag, it'd keep this additional logic out of the function.

Reply:
we have 2 promo strings, i think it's over complicated to pass them both. And a little weird to pass only one